research/README.md

```diff
@@ -0,0 +1,325 @@
+
+| Empty state | Explore a cid | Drill into a data node |
+|-|-|-|
+|![](img/004-ipld-explorer-webui-1.png) |![](img/004-ipld-explorer-webui-2.png) |![](img/004-ipld-explorer-webui-3.png)
+
+The UI/UX should be improved but the idea is interesting, and could become a vital tool for helping people to understand IPFS and IPLD.
+
+## Pinning
+
+Discussion: https://github.com/ipfs-shipyard/pm-ipfs-gui/issues/10
+
+**Desktop** allows you to pin a CID. You don't need to aleady store that content. You can paste in any CID and give it a label to remind yourself what it was.
+
+| Desktop pin pane | Add a pin |
+|------------------|-----------|
+| ![](img/005-pin-desktop-1.png) | ![](img/005-pin-desktop-2.png) |
+
+| Filled out form | PIN in progesss
+|-----------------|------------------|
+| ![](img/005-pin-desktop-3.png) | ![](img/005-pin-desktop-4.png) |
+
+## Open WebUI
+
+Both Desktop and Companion offer this, but is "Open WebUI" the right name for this action? As a companion or desktop user, I may have never seen the WebUI app, and so may have no intuitions as to what it's for.
+
+Desktop and companion have differing ideas as to what the best url for WebUI is.
+
+desktop:
+http://localhost:5001/ipfs/QmPhnvn747LqwPYMJmQVorMaGbMSgA7mRRoyyZYz3DoZRQ/#/
+
+ipfs companion
+http://127.0.0.1:5001/ipfs/QmPhnvn747LqwPYMJmQVorMaGbMSgA7mRRoyyZYz3DoZRQ/#/
+
+## IPFS Node Config vs Settings / Open Preferences
+
+Discussion: https://github.com/ipfs-shipyard/pm-ipfs-gui/issues/15
+
+**WebUI** allows you to modify the IPFS node config.
+
+**Desktop** and **Companion** have to deal with app specific settings and also let the user modify the IPFS node config as well.
+
+We need to think about how that comes across to the user. We can keep app settings and node config separate, but we should aim to present them as sections of the same page in the UI.
+
+## Start / Stop IPFS Node
+
+Desktop let's you start and stop your IPFS node. Companion and WebUI don't
+
+In Desktop the control is shown as a the `Power` icon, bottom right. It's not clear that that is it's purpose. I initially thought it was a fancy quit this app button.
+
+<img width=600 title='IPFS Desktop with a stopped IPFS node' src='img/desktop-node-stopped.png' />
+
+Using it fades out the node info, but it isn't clear what's happened, or that toggling the same button will bring it back to life. The other tabs show no indication that the node is stopped, and attempting to add a file causes an error.
+
+
+## Control which IPFS node you connect to
+
+Companion let's you toggle between an embedded js-ipfs node or an external IPFS daemon. In the preferences you can even have it connect to a remote IPFS node.
+
+Desktop uses [`js-ipfsd-ctl`](https://github.com/ipfs/js-ipfsd-ctl) and bundles `go-ipfs`, and it's not configureable. If you run a local ipfs daemon, Desktop cannot start, and instead shows an error message, "Please stop your IPFS daemon before running this application".
```

Comment:
typo: `configureable` → `configurable`

Reply:
May be worth linking related issues:
- https://github.com/ipfs-shipyard/ipfs-desktop/issues/542
- https://github.com/ipfs-shipyard/ipfs-desktop/issues/611